packages/tailwindcss-language-server/src/util/v4/design-system.ts

```diff
@@ -37,7 +37,11 @@ export async function loadDesignSystem(
   let resolved = await resolveCssImports().process(css, { from: filepath })
 
   // Step 3: Take the resolved CSS and pass it to v4's `loadDesignSystem`
-  let design = tailwindcss.__unstable__loadDesignSystem(resolved.css) as DesignSystem
+  let design: DesignSystem = await tailwindcss.__unstable__loadDesignSystem(resolved.css, {
+    loadPlugin() {
+      return () => {}
```

Comment:
We're intentionally not handling `@plugin` here — this PR is purely to make sure alpha 19 doesn't break in intellisense. That support will be done in a separate PR.